localMatchApp/Sources/logIn.cpp

```diff
@@ -0,0 +1,71 @@
+#include "logIn.h"
+#define OCUPATED_USER 3
```

Comment:
these error messages would be better to be contained in an enum

Reply:
Done